tests/dummy/app/routes/application.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  intl: Ember.inject.service(),
+  beforeModel() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).